public/js/index.js

```diff
@@ -979,6 +979,25 @@ function initRepository() {
     }
 }
 
+var toggleMigrations = function(){
+    var authUserName = $('#auth_username').val();
+    var cloneAddr = $('#clone_addr').val();
+    if (!$('#mirror').is(":checked") && (authUserName!=undefined && authUserName.length > 0) 
+    && (cloneAddr!=undefined && (cloneAddr.startsWith("https://github.com") || cloneAddr.startsWith("http://github.com")))) {
```

Comment:
Should it be mentioned that the checkboxes only show up after you add some clone auth?  
  
Personally, I was confused at first since I planned on migrating public repos that otherwise would not have needed auth.  
After looking, it makes sense that these credentials are needed for the API regardless of the privacy of the repo, however I think it might be beneficial to notify users of that somewhere.

Otherwise, users may migrate their public repos and never even know this is possible since the checkboxes are completely hidden otherwise.